samples/bot-file-upload/python/appManifest/manifest.json

```diff
@@ -25,7 +25,7 @@
   "accentColor": "#abcdef",
   "bots": [
     {
-      "botId": "<<MICROSOFT-APP-ID>>",
+      "botId": "${{AAD_APP_CLIENT_ID}}",
```

Comment:
Need to change the holder string described in section Run the app (Manually Uploading to Teams) of README.md too.

Please apply this for all 4 samples.

Reply:
I have updated all the placeholders, such as `{{AAD_APP_CLIENT_ID}}`.